nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const app = express();
+module.exports = app;
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+const mealsNum = meals.map((meal) => meal.id);
+const reviewMealId = reviews.map((review) => review.mealId);
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("asd");
+});
+// meals with review
+app.get("/meals", async (request, response) => {
+  const mealWithReview = meals.map(meal => {
+    const reviewMatchMeal = reviews.filter(review => review.mealId === meal.id)
+    return {
+      ...meal,
+      reviews: reviewMatchMeal
+    }
+  })
+  response.send(mealWithReview);
```

Comment:
Prefer to use `response.json` when sending back JSON data. 
Check [this link](https://www.tutorialspoint.com/difference-between-res-send-and-res-json-in-express-js#:~:text=send%20function%20sets%20the%20content,now%20treat%20it%20as%20text.&text=json%20function%20on%20the%20other,the%20response%20to%20the%20client.) for details.

Reply:
got it now, Tks.